beacon/blockchain/finalize_block.go

```
@@ -181,8 +183,7 @@ func (s *Service) executeStateTransition(
 		WithVerifyPayload(true).
 		WithVerifyRandao(false).
 		WithVerifyResult(false).
-		WithMeterGas(true).
-		WithOptimisticEngine(true)
```

Dropped `WithOptimisticEngine` since we don't silence any error anymore